xml/hwbp_registers.xml

```diff
@@ -967,6 +967,15 @@
         tie any number of upper bits to 0. The $|select|$ bits may only support
         0 (ignore).
 
+        Byte-granular comparison of the \RcsrScontext to \FcsrTextraThirtytwoSvalue
+        allows \RcsrScontext to be defined to include more than one element of comparison.
+        For example, software instrumentation can program the \RcsrScontext value to be
+        the concatenation of different ID contexts such as process ID and thread ID.
+        The user can then program byte compares to include one or more of the contexts
+        in the compare. For \RcsrTextraThirtytwo there are two byte-compare mask bits.
```

Comment:
No need to talk explicitly about textra32 because this paragraph is in the section that defines textra32, and the size is already clear from the figure that is placed directly below this block of text.